### PR TITLE
Fixed #2186: Standardize tab theming

### LINKED ISSUE #2186: Night mode: Tabs
**Summary:** 

Tabs are inconsistent in how they are styled. I'm okay with MainActivity being different but Search and Category tabs should definitely look the same.

Should also probably standardise across non-night mode too.

Tabs are used in (edit this list if I've forgotten some):
- MainActivity (Contributions/Nearby)
- SearchActivity (Media/Categories)
- Category page (Media/Subcategories/Parent categories)
- Bookmarks page (Pictures/Locations)

**Screen-shots:** 

| MainActivity | SearchActivity | Category |
| - | - | - |
| screenshot_1545301973 | screenshot_1545302017 | screenshot_1545302049 |

### PR DESCRIPTION
**Pull request by borisenkotim**
**Description (required)**

Changed the color of tabs of the bookmarks page, to standardize and look the same as categories page
Update: Also changed the search activity page 

Fixes #2186: Night mode: Tabs

What changes did you make and why?

The only real changes happened in the activity_bookmarks.xml and activity_search.xml files. The other files(searchActivity, settingsActivity) show up as changes because because I was running a few tests but have since changed the code back to its original form, and when I ran the emulator (or updated sdk tools) android studio automatically applied a change to the build.gradle file from buildToolsVersion buildToolsVersion to buildToolsVersion '28.0.3' for some reason

**Tests performed (required)**

Testing done on LG V30 (direct) With API level 25
Testing done on nexus 5x emulator API level 28

---

_Note: Please ensure that you have read CONTRIBUTING.md if this is your first pull request._

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -226,7 +226,7 @@ android {
         targetCompatibility JavaVersion.VERSION_1_8
     }
 
-    buildToolsVersion buildToolsVersion
+    buildToolsVersion '28.0.3'
```
Comment (maskaravivek) [inline app/build.gradle]: Undo this change

**Comment by domdomegg** [comment]
Sorry for the long wait on reviewing this.

Tested on Galaxy Nexus, API level 28:

Screenshot_1552677805
Screenshot_1552677797
Screenshot_1552677791
Screenshot_1552677786
Screenshot_1552677714
Screenshot_1552677695
Screenshot_1552677673
Screenshot_1552677669
